examples/platform/silabs/silabs_utils.h

```diff
@@ -24,6 +24,8 @@
 extern "C" {
 #endif
 
+bool isLogInitialized(void);
```

Comment:
Never realized how silabsInitLog and silabsLog were done. 
Can't say I like it with he extern.

Can we move this header to `src/platform/silabs/ a`nd include it in 
`src/platform/silabs/Logging.cpp`

Reply:
Created Logging.h for this, will do a cleanup in a different PR.